### ISSUE TITLE
[DataGrid] Add the ability to hide the operators column of the filter panel

### ISSUE THREAD

**Issue by razk87** [mui/mui-x#2740]
### Order ID 💳

#29674

### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Summary 💡

I'm trying to disable the operators functionality from a colum filter,
so the default will be isEqual/ is true,  and the operator drop down will not be visible at all

my expacted result is like this image below:
![Group 66618 (1)](https://user-images.githubusercontent.com/83636963/135312341-48c4a1c2-5d00-4c8e-8625-c59a715d78e8.jpeg)

### Your environment 🌎

<details>
  <summary>`npx @material-ui/envinfo`</summary>

```
  Don't forget to mention which browser you used.
  Output from `npx @material-ui/envinfo` goes here.
```
</details>

**Comment by flaviendelangle** [comment]
Not that I am aware of
You can override the whole `FilterPanel` component with the `components` props.
But I would require to re-implement a log of things.

Maybe we should hide the `operator` column when their is only one operator.

**Comment by m4theushw** [comment]
> Maybe we should hide the operator column when their is only one operator.

If the only operator is "contains", then it's fine to hide as the user can treat it like a normal search filter. The problem comes if the hidden operator is "equals", or worse with "starts with" (a edge case), since it's not obvious what is being filtered.

We could add another `FilterForm` slot to allow to override only the filter row, not the entire panel.

**Comment by flaviendelangle** [comment]
If we start to hide it based on which operator remains I think it will introduce to much confusion.

A `FilterForm` slot would work (I would rename it `FilterItemForm` to clarify that it only handles a single filter

**Comment by m4theushw** [comment]
Similar to https://github.com/mui-org/material-ui-x/issues/3082#issuecomment-983683118, a new prop could be added to `GridFilterPanel` to hide the operators when there's only one option. A new slot for `GridFilterForm` and adding this prop on it could also be an option.

**Comment by flaviendelangle** [comment]
@alexfauquette I think #3384 closes this one with `linkOperators={[]}`

**Comment by alexfauquette** [comment]
It's fixed by #3497

As suggested by @flaviendelangle https://github.com/mui/mui-x/issues/2740#issuecomment-932097852 you can force to have only one operator.
You can hide the operator by [passing props to the filter panel](https://mui.com/components/data-grid/filtering/#customize-the-filter-panel-content)

```jsx
<DataGridPro
	  componentsProps={{
	      filterPanel: {
	          filterFormProps: {
	              operatorInputProps: {
	                  disabled: true, // If you only want to disable the operator
	                  sx: { display: "none" } // If you want to remove it completely
	              },
	          }
	      }
	}}
	// ...
/>
```
